libsession/src/main/java/org/session/libsession/messaging/sending_receiving/MessageSenderClosedGroupHandler.kt

```diff
@@ -161,7 +161,7 @@ fun MessageSender.addMembers(groupPublicKey: String, membersToAdd: List<String>)
     for (member in membersToAdd) {
         val closedGroupNewKind = ClosedGroupControlMessage.Kind.New(ByteString.copyFrom(Hex.fromStringCondensed(groupPublicKey)), name, encryptionKeyPair, membersAsData, adminsAsData, expireTimer)
         val closedGroupControlMessage = ClosedGroupControlMessage(closedGroupNewKind)
-        closedGroupControlMessage.sentTimestamp = sentTime
+        closedGroupControlMessage.sentTimestamp = System.currentTimeMillis()
```

Comment:
Can you explain the idea behind this change?

Reply:
It is to have 2 different timestamps for the MEMBERS ADDED update and the NEW update. On the receiving side I noticed after the member receives the NEW update, he then pulls the MEMBERS ADDED one. Having a different timestamp here allows to ignore that notification in isValidGroupUpdate as we don't need to process it.